docs/source/_static/data_formatting_examples/exec_format_structured_data.py

```diff
@@ -0,0 +1,151 @@
+"""
+Format Structured Data
+======================
+
+Format a dataset where the filenames of the tiles provide most of the metadata required to
+organize them into the 5D set (image_type, fov, round, ch, zplane) of 2D tiles  (y, x).  The
+remaining metadata, namely physical coordinates of the tiles, are provided in a CSV file.
+
+The stem of the tiles' filenames must be
+`<image_type>-f<fov_id>-r<round_label>-c<ch_label>-z<zplane_label>`.  The extension should be one
+of the supported tile formats (:py:class:`~slicedimage.ImageFormat`).  For example, the file
+`nuclei-f0-r2-c3-z33.tiff` would belong to the nuclei image, fov 0, round 2, channel 3, zplane 33.
+
+The CSV file must have a header, and must contain the following columns:
+
+====== ======== ====================================================================================
+Column Required Notes
+------ -------- ------------------------------------------------------------------------------------
+fov    yes
+round  yes
+ch     yes
+zplane yes
+xc_min yes      This should be the minimum value of the x coordinate of the tile.
+xc_max yes      This should be the maximum value of the x coordinate of the tile.
+yc_min yes      This should be the minimum value of the y coordinate of the tile.
+yc_max yes      This should be the maximum value of the y coordinate of the tile.
+zc_min no       This should be the minimum value of the z coordinate of the tile.
+zc_max no       This should be the maximum value of the z coordinate of the tile.
```

Comment:
if someone provides only the max_z, does this fail?

Reply:
It does not fail.  If both `zc_min` and `zc_max` are set, we treat it as a range.  If `zc_min` xor `zc_max` are set, then we treat it as a scalar.

Do you think that is wrong?

Reply:
Nope, that's perfect. :+1: